pkgs/development/python-modules/alectryon/c779def3fa268e703d4e0ff8ae0b2981e194b269.patch

```diff
@@ -0,0 +1,37 @@
+From c779def3fa268e703d4e0ff8ae0b2981e194b269 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Cl=C3=A9ment=20Pit-Claudel?= <clement.pitclaudel@live.com>
+Date: Tue, 17 Aug 2021 11:29:16 -0400
+Subject: [PATCH] cli: Don't copy permissions of asset files
+
+* alectryon/cli.py (copy_assets): Use shutil.copyfile (which does not copy file
+permissions) instead of shutil.copy.
+
+https://github.com/NixOS/nixpkgs/pull/134454
```

Comment:
Can we fetch this PR instead?

Reply:
@SuperSandro2000: this PR is just the place where I reported the issue to the upstream maintainer.

Reply:
did you submit it to the alectryon project? Then we could fetch that commit? If not please name the patch better than the commit sha.

Reply:
The commit was not authored by myself but by the project maintainer. So yes, we can fetch the commit from the project if that's considered better practice.

Reply:
Yes, please do that.

Reply:
Done.